docs/modules/mongodb_watchdog.md

```diff
@@ -1,8 +1,14 @@
 # Logger: `mongodb_watchdog`
 
-The `mongodb_watchdog` module stores the watchdog entries for your Drupal site in MongoDB collection as mentioned in [Settings Configuration](../../install#settings-configuration)
+The `mongodb_watchdog` module stores the log entries for your Drupal site
+in MongoDB collections as mentioned in [Settings Configuration](../../install#settings-configuration)
 
-This module uses a separate database, using the `logger` database alias in settings. Do NOT point that alias to the same database as `default`, because the module drops the `logger` database when uninstalling, which would drop all your other data with it.
+This module uses a separate database, defined by the `logger` database alias in settings.
+Do NOT point that alias to the same database as `default`, because the module
+drops the `logger` database when uninstalling, which would drop all your other data with it.
+
+## Configuration
+  Below are the configurable variables available in `mongodb_watchdog.settings`
```

Comment:
Drupal 8 no longer has "variables": better avoid the term, which makes one think this is unadapted D7 doc. AIUI the term for this is `configuration entry` (or is it `item` , or `value` ?) with `mongodb_watchdog.settings` being a configuration entity.